apollo-router/tests/integration/typename.rs

```diff
@@ -71,6 +71,72 @@ async fn aliased() {
     "###);
 }
 
+/* FIXME: should be fixed in query planner, failing with:
+   > value retrieval failed: empty query plan. This behavior is unexpected and we suggest opening an issue to apollographql/router with a reproduction.
```

Comment:
Can't be fixed in scope of this PR since they fail on query planning

Reply:
I will open an issue right now.
Probably open it for the router and see if it should be fixed in router or in federation.

Reply:
I open an issue

Reply:
I’d prefer not to have entire functions or tests commented out in the repo. Please use `#[ignore]` or `#[should_panic]` instead. Also please link to the issue in the FIXME comment